wandb/sdk_py27/wandb_run.py

```diff
@@ -1769,6 +1775,95 @@ def _save_job_spec(self):
             print(s, file=f)
         self.save(spec_filename)
 
+    def define_metric(
+        self,
+        name,
+        step_metric = None,
+        step_sync = None,
+        hide = None,
+        summary = None,
+        goal = None,
+        overwrite = None,
+        **kwargs
+    ):
+        """Define metric properties which will later be logged with `wandb.log()`.
+
+        Arguments:
+            name: Name of the metric.
+            step_metric: Independent variable associated with the metric.
+            step_sync: Automatically add `step_metric` to history if needed.
+            hide: Hide this metric from automatic plots.
+            summary: Specify aggregate metrics added to summary.
+                Supported aggregations: "min,max,mean,best"
```

Comment:
It wasn't clear to me how `best` was any different than min or max.  We should probably say something here about **best** defaulting to min if goal isn't specified or just say the default goal is minimize..